src/main/resources/com/adobe/jenkins/github_pr_comment_build/TriggerPRLabelBranchProperty/help-commentBody.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    The Label to trigger a new build for a PR job when it is received.
```

Comment:
Can you change this to `The regex label value to check against if a new build should be triggered or not`

Reply:
Done